### PR TITLE
Add menu bar for future frameless mode, add attempt quit with dialog …

### LINKED ISSUE #8: Dialog before launcher exit
If the user attempts to close the launcher, it will instaquit all launched interface processes, so let's make it so that a dialog shows warning of this (if an instance is running) and asks "Are you sure" or something of the sort.

### PR DESCRIPTION
**Pull request by two-one-five**
…if interface running.

### CODE REVIEW

**Comment by Nex-Pro** [comment]
@kasenvr can we get a name for the color on line 14 of App.vue ?

**Comment by Nex-Pro** [comment]
https://www.colorhexa.com/182b49

**Comment by two-one-five** [comment]
Do you mean to use a name instead of a hex? I scrolled through the list of CSS names detected by Chrome and none of them really came close to this color sadly.

**Comment by Nex-Pro** [comment]
Even put a descriptive comment might be good. HiFi named colors. It saves having the reader try to work out what the color is if it has a name or a description.

**Comment by ctrlaltdavid** [comment]
README.md needs updating with instructions on how to install tasklist module.

File: src/components/Dialogs/WantToClose.vue [hunk 1]
```diff
@@ -0,0 +1,71 @@
+<!--
+//  WantToClose.vue
+//
+//  Created by David Rowe on 14 Feb 2020.
```
Comment (ctrlaltdavid) [inline src/components/Dialogs/WantToClose.vue]: No I didn't.

File: src/components/Dialogs/WantToClose.vue [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+const { ipcRenderer } = require('electron');
+
+export default {
+    name: 'WantToClose',
+	methods: {
+        continueClose: function () {
+            ipcRenderer.send('close-launcher');
+        },
+        cancelClose: function () {
+            // nothing to do.
```
Comment (ctrlaltdavid) [inline src/components/Dialogs/WantToClose.vue]:
```suggestion
            // Nothing to do.
```

File: src/components/Dialogs/WantToClose.vue [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        },
+     },
+    data: () => ({
+        showWantToClose: true,
+    }),
+};
+</script>
```
Comment (ctrlaltdavid) [inline src/components/Dialogs/WantToClose.vue]: Missing new line at end of file.

**Review by ctrlaltdavid — CHANGES_REQUESTED** [review]

**Review by ctrlaltdavid — APPROVED** [review]

**Pull request merged**